emsymbolizer.py

```diff
@@ -218,12 +218,19 @@ def symbolize_address_sourcemap(module, address, force_file):
 
 def symbolize_address_symbolmap(module, address, symbol_map_file):
   """Symbolize using a symbol map file."""
+
+  def split_symbolmap_line(line):
+      index = line.find(':')
```

Comment:
I think you can just do `return line.split(':', 1)`

If you want and explicit error you could also add `assert ':' in line, 'f'invalid symbolmap line: {line}'`